cli/app/internal/runtimestate/runtime_events.go

```
@@ -364,6 +438,62 @@ func containsQueuedUserMessageID(messages []clientui.QueuedUserMessage, id strin
 	return false
 }
 
+func removePendingQueuedUserMessageByID(messages *[]clientui.QueuedUserMessage, id string) bool {
```

golangci-lint (2.12.2)

[error] func removePendingQueuedUserMessageByID is unused

(unused)